tests/unit/participants/test_models.py

```diff
@@ -0,0 +1,42 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from participants.models import Participant, ParticipantTeam
+
+
+class BaseTestCase(TestCase):
```

Comment:
Can you please be consistent with your use of single and double quotes everywhere. We however prefer single quotes.